packages/itwin/property-grid/package.json

```diff
@@ -29,28 +29,39 @@
     "copy:public": "cpx \"./public/**/*\" ./lib/public",
     "copy:cjs": "cpx \"./src/**/*.scss\" ./lib/cjs",
     "copy:esm": "cpx \"./src/**/*.scss\" ./lib/esm",
-    "cover": "",
+    "cover": "nyc npm run test",
+    "extract-api": "betools extract-api --entry=property-grid-react",
     "lint": "eslint -f visualstudio \"./src/**/*.{ts,tsx}\" 1>&2",
     "lint:fix": "npm run lint -- --fix",
     "pseudolocalize": "betools pseudolocalize --englishDir ./public/locales/en --out ./lib/public/locales/en-PSEUDO",
-    "test": "",
+    "test": "mocha \"./lib/cjs/test/**/*.test.js\"",
     "clean": "rimraf lib .rush/temp/package-deps*.json",
     "rebuild": "npm run clean && npm run build"
   },
+  "peerDependencies": {
+    "@itwin/appui-abstract": "^4.0.0",
+    "@itwin/appui-react": "^4.0.0",
```

Comment:
should appui peers be ^4.1 to match the devDeps?

Reply:
No there is no need to bump peerDeps at the moment.